components/collection/unlockable/UnlockableContainer.vue

```diff
@@ -298,12 +298,16 @@ const handleSubmitMint = async () => {
       UNLOCKABLE_CAMPAIGN
     ).then((res) => {
       toast('mint success')
-      justMinted.value = `${collectionId}-${res.result.sn}`
       scrollToTop()
+      return `${collectionId}-${res.result.sn}`
     })
+    // 20s timeout
+    setTimeout(() => {
+      isLoading.value = false
+      justMinted.value = id
+    }, 40000)
```

Comment:
This is not 20 sec 😃 , what about your solution to fetch data using polkadot api?

Reply:
> what about your solution to fetch data using polkadot api?

This should have been done a lot of hours ago ^-^. Spend the night debugging fancy stuff. Will refactor after decoded